serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -8,7 +8,7 @@
 :grpc_core_url: https://grpc.io/docs/what-is-grpc/core-concepts
 :grpc_json_mapping: https://developers.google.com/protocol-buffers/docs/proto3#json
 :grpc_configuration_url: https://quarkus.io/guides/grpc-service-consumption#client-configuration
-:grpc_enum_url:https://developers.google.com/protocol-buffers/docs/proto3#enum
+:grpc_enum_url: https://developers.google.com/protocol-buffers/docs/proto3#enum
```

Comment:
unrelated?

Reply:
uber small fix, that's why I've added here.